cabal-dev-scripts/src/GenUtils.hs

```diff
@@ -162,6 +167,7 @@ toConstructorName t = t
     special :: Text -> Text
     special "0BSD"          = "NullBSD"
     special "389_exception" = "DS389_exception"
+    special "3D_Slicer_1_0" = "X3D_Slicer_1_0"
```

Comment:
Ok, after a chat with Hécate, we decided to go back to the usual “exception” method, since the release is time-sensitive and there is no room for mistakes.

Reply:
Makes sense.